schemas/preferences.sublime-settings.json

```diff
@@ -834,6 +894,11 @@
           "type": "boolean",
           "default": true
         },
+        "regex_auto_escape": {
+          "markdownDescription": "If true, patterns copied from selected text to the find panel\nwill automatically be escaped when doing a regex search.",
```

Comment:
Shouldn't have newline here. We should let the popup width limit the width.

Reply:
This is e.g. such a setting which is available in latest dev builds, but users on current stable build might wonder why it doesn't work when being suggested.